client/source/components/overview/sub-components/ProductStyles.jsx

```diff
@@ -0,0 +1,35 @@
+
+// LIBRARY IMPORTS
+import React from 'react';
+
+import StyleButton from './StyleButton.jsx';
+
+
+// MAIN
+const ProductStlyes = function CreateProductStlyesComponent({
```

Comment:
Your code always looks so good, I don't have much to say, but here you misspelled "Stlyes"

Reply:
Brett always catching those misspellings 😂